09b_Custom_Coadd_Sources.ipynb

```diff
@@ -0,0 +1,1248 @@
+   "metadata": {},
+   "source": [
+    "Store the TAP results in a Pandas table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3fd7700a-f2e5-42cd-aa30-3b427540be0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tap_table = tap_results.to_table().to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8571e3d-836a-46bd-bd0d-067236970ccb",
+   "metadata": {},
+   "source": [
+    "Convert Object sky coordinates (RA, Dec) to deepCoadd pixels (x,y) and store in the TAP results table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e670d9c6-c9d9-498c-9d90-64dc6b8ab254",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bbox = cutout.getBBox()\n",
+    "wcs = cutout.getWcs()\n",
+    "temp1 = np.zeros(len(tap_table), dtype='float')\n",
+    "temp2 = np.zeros(len(tap_table), dtype='float')\n",
+    "\n",
+    "for i in range(len(tap_table['coord_ra'].values)):\n",
+    "    sP = lsst.geom.SpherePoint(tap_table['coord_ra'][i]*lsst.geom.degrees, tap_table['coord_dec'][i]*lsst.geom.degrees)\n",
+    "    cpix = wcs.skyToPixel(sP)\n",
+    "    temp1[i] = float(cpix[0])\n",
+    "    temp2[i] = float(cpix[1])\n",
+    "    del sP, cpix\n",
+    "\n",
+    "tap_table['cutout_x'] = temp1\n",
+    "tap_table['cutout_y'] = temp2\n",
+    "\n",
+    "del bbox, wcs, temp1, temp2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4be8348c-1c0c-4280-92ad-6cc80c34cde3",
+   "metadata": {},
+   "source": [
+    "### 3.3.3. Plot images side-by-side\n",
+    "\n",
+    "Plot the new and original cutouts side-by-side, with detected sources (orange) and Objects (red)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc37b989-2beb-4e87-bf08-a0afe0f1c1f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(1, 2, figsize=(14, 7), sharey=False, sharex=False)\n",
+    "\n",
+    "plt.subplot(1, 2, 1)\n",
+    "disp1 = afwDisplay.getDisplay(frame=fig)\n",
+    "disp1.scale('asinh', 'zscale')\n",
+    "disp1.mtv(my_cutout.image)\n",
+    "\n",
+    "with disp1.Buffering():\n",
+    "    for s in sources:\n",
+    "        if((s.getX() > 11900) & (s.getX() < 12500)\n",
+    "           & (s.getY() > 7900) & (s.getY() < 8400)):\n",
+    "            disp1.dot('o', s.getX(), s.getY(), size=10, ctype='orange')\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "disp2 = afwDisplay.getDisplay(frame=fig)\n",
+    "disp2.scale('asinh', 'zscale')\n",
+    "disp2.mtv(cutout.image)\n",
+    "\n",
+    "with disp2.Buffering():\n",
+    "    for s in range(len(tap_table)):\n",
+    "        if((tap_table['cutout_x'][s] > 11900) & (tap_table['cutout_x'][s] < 12500)\n",
+    "           & (tap_table['cutout_y'][s] > 7900) & (tap_table['cutout_y'][s] < 8400)):\n",
+    "            disp2.dot('o', tap_table['cutout_x'][s], tap_table['cutout_y'][s], size=10, ctype='red')\n",
+    "\n",
```

Comment:
Add axis labels and titles to these images? (e.g., "Window 1 cutout" and "deep cutout"?)

Reply:
Added titles.

Reply:
Ugh it worked for a hot second and now it doesn't and I can't figure it out. Minor, leaving for another time. Added more figure description to compensate.